e2e/test_operator/operator_test.go

```diff
@@ -1428,6 +1429,50 @@ var _ = Describe("Operator", Label("e2e", "pr"), func() {
 		})
 	})
 
+	// this test took 14min to run for me, so I think it is best to just have the one test
+	When("replacing Pods due to securityContext changes", func() {
+		var initialPods *corev1.PodList
+		var originalPodSpec, modifiedPodSpec *corev1.PodSpec
+
+		BeforeEach(func() {
+			if fdbCluster.GetClusterSpec().ReplaceInstancesWhenResourcesChange != nil {
+				Expect(*fdbCluster.GetClusterSpec().ReplaceInstancesWhenResourcesChange).To(BeFalse())
+			}
+			originalPodSpec = fdbCluster.GetPodTemplateSpec(fdbv1beta2.ProcessClassStorage)
```

Comment:
Using `General` will be faster as there will be no data movement involved.

Reply:
yeah I was working under the assumption that PVC should be a criteria for the change